jinja/carnival/community_calls.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %}
+{% block content %}
+{% block navbar %}
+{% include "navbar.html" %}
+{% endblock %}
+
+<div class="parallax" style="background-image: url('../img/xanadu-background.png'); box-shadow: none;">
+    <div class="container">
+
+        <div class="mt-5 pt-5 text-center teal-text text-lighten-2">
+            <img src="../img/carnival_logo_splash.png" class="logo"></img>
+        </div>
+        <div class="row text-center">
+            <h3 class="col-12 mt-5 mb-5 grey-text" style="font-family: 'Roboto', sans-serif !important;">
+            The PennyLane <strong>Community Calls</strong> are an important gathering place for us!
+            </h3>
+	<div style="font-size: 14pt;">
+            <p>
+            Our Thursday community calls are here for you to ask questions, learn something new, 
+            meet with other community members and much more. 
+            Come every Thursday at 11am ET to the community-call voice channel of the <a href ="https://discord.com/invite/JqVGmpkP96">Unitary Fund Discord</a>.
+
+</p>
+<p>
+    <style>.myButton {background-color:#44c767;
+        border-radius:28px;border:1px solid #18ab29;
+        display:inline-block;
+        cursor:pointer;color:#ffffff;
+        font-family:Arial;font-size:16px;
+        padding:16px 31px;text-decoration:none;
+        text-shadow:0px 1px 0px #2f6627;}.myButton:hover 
+        {background-color:#5cbf2a;}.myButton:active {position:relative;top:1px;}</style>
+        <a href="https://discord.com/invite/JqVGmpkP96" class="myButton">Join the Discord here!</a>
+
+</p>
+<p>
+Our next community call is on <strong>November 11th at 11am ET</strong>. On this occasion we will talk about some of the 
+best new features in the latest PennyLane release. We will also learn about a very important deprecation 
+so don't miss the call!
+</p>
```

Comment:
```suggestion
```
Not suitable for a static webpage (date has even passed already!)

Reply:
Couldn't we add the next one and change it next Thursday?